grommet/grommet
# Gets the remove icon from theme

#### What does this PR do?
Allows the remove icon in `Tag` component to be provided from the theme. Fixes #6870 

#### Where should the reviewer start?
Start with the `Tag.js` file

#### What testing has been done on this PR?
1. Ran unit tests and failures
2. Updated snapshot
3. Manually tested in storybook by updating the icon from theme and is working fine.

#### How should this be manually tested?
1. Run storybook and navigate to `Tag > onRemove` story
2. Verify the `FormClose` icon is rendered by default
3. Update the icon(`tag.icons.remove`) from `themes > base.js` and see the changes reflected on the story
4. Repeat the same with `Tag > Size` story as well

#### Do Jest tests follow these best practices?
~~Not applicable as no new tests are added~~ Added a new test.


#### Any background context you want to provide?

#### What are the relevant issues?
#6870 

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
Yes. https://v2.grommet.io/tag#theme

#### Should this PR be mentioned in the release notes?
Yes.

#### Is this change backwards compatible or is it a breaking change?
This is backward compatible
